libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SmartStorePlugin.java

```diff
@@ -59,25 +59,11 @@
 public class SmartStorePlugin extends ForcePlugin {
 
 	// Keys in json from/to javascript
-	public static final String BEGIN_KEY = "beginKey";
```

Comment:
Just curious @wmathurin if it's ok to remove all those keys in terms of upgrading..the change should not impact upgrading right?